fast_forward/interaction_plots.py

```diff
@@ -0,0 +1,48 @@
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pickle
+
+def make_distribution_plot(data, out, atom_list, interaction, plot_data=False):
+
+    x = data.T[0]
+    y = data.T[1]
```

Comment:
same as above:

- give it an ax argument
- give it a save argument
- return the axis object